src/index.js

```diff
@@ -299,13 +300,33 @@ export default class AuthnWidget {
   }
 
   postDeviceProfileAction() {
-    let profilingElement = document.querySelector('[data-profilingtype]');
-    switch (profilingElement.dataset['profilingtype']) {
+    let data = this.store.getStore();
+    switch (data.deviceProfilingType) {
       case 'IDW':
+      case 'TMX-WEB':
+        var script = document.createElement('script');
+        script.src = data.deviceProfilingScriptUrl;
+        document.head.appendChild(script);
+
+        setTimeout(() => {
+          this.store.dispatch('POST_FLOW', 'continueAuthentication', '{}');
+        },
+        parseInt(data.deviceProfilingTimeoutMillis));
```

Comment:
Can you format this setTimeout function? I was confused because it reads like the parseInt is not part of the setTimeout function.

Reply:
Formatted.